src/ec/packages/ec-component-file-upload/file-upload.story.js

```diff
@@ -0,0 +1,38 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text, boolean, object } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+// Import data for tests
+import dataDefault from '@ecl/ec-specs-file-upload/demo/data--default';
+
+import fileDocs from './docs/file-upload.md';
+import fileUpload from './file-upload.html.twig';
+
+storiesOf('Components/Forms/File Upload', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withNotes)
+  .addDecorator(withCode)
+  .add(
+    'default',
+    () =>
+      fileUpload({
+        ...dataDefault,
```

Comment:
Not sure you really need the ...dataDefault here for the way you are setting the story, all the properties are redefined later.

Reply:
ok